### PR TITLE
add custom static functions, remove preloaders and small cleanup

### PR DESCRIPTION
**Pull request by czue**
commit-by-commit

Not sure who is best to review this. Possibly @ctsims @wpride or @benrudolph? cc @kaapstorm 

This just opens up the possibility for HQ to do this. Won't be useful until http://manage.dimagi.com/default.asp?166374 is addressed

### CODE REVIEW

File: touchforms/backend/xformserver.py [hunk 1]
```diff
@@ -365,6 +367,5 @@ def main(port=DEFAULT_PORT, stale_window=DEFAULT_STALE_WINDOW, ext_mod=[], offli
 
     main(
         port=options.port,
-        stale_window=options.stale_window,
-        ext_mod=args
```
Comment (czue) [inline touchforms/backend/xformserver.py]: only bhoma was using the command line extensions and bhoma has long since forked from being supported on master

**Comment by czue** [comment]
this PR kept changing. just pinging people in case the edit on the description didn't notify you:

commit-by-commit

Not sure who is best to review this. Possibly @ctsims @wpride or @benrudolph? cc @kaapstorm

This just opens up the possibility for HQ to do this. Won't be useful until http://manage.dimagi.com/default.asp?166374 is addressed

**Comment by benrudolph** [comment]
looks good to me, will wait for others to review (unless you'd like it merged sooner)

**Comment by czue** [comment]
Not sure if @wpride or @ctsims is going to review this. @gcapalbo I also ran the inbound_handlers test, but let me know if there's anything else I should do.

failing that, @benrudolph I think is probably good to go?

**Comment by gcapalbo** [comment]
@czue the only other test to run besides inbound_handlers are the kookoo tests (requires same setup as inbound handlers and just uncomment the tests from kookoo/tests/__init__.py)

**Comment by czue** [comment]
cool, those also passed.

**Comment by wpride** [comment]
@czue looks good to me

**Comment by benrudolph** [comment]
cool let's merge this bad boy

**Pull request merged**